file/file-api/src/main/java/uk/gov/justice/services/file/api/requester/StreamingFileRequester.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.services.file.api.requester;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+public interface StreamingFileRequester extends FileRequester{
+    /**
```

Comment:
not sure if extending the FileRequester interface is a good idea. Maybe we should keep those separate and make Alfresco implement both of them for the time being.